aii-pxelinux/src/main/perl/pxelinux.pm

```diff
@@ -67,23 +78,40 @@ sub get_anaconda_version
     return $version;
 }
 
-# Returns the absolute path where the PXE file must be written.
+# Prints an IP address in hexadecimal.
+sub hexip
```

Comment:
preferably, you have a method per variant, and handle the variant selection like the format_method.